support/docker/production/Dockerfile.stretch

```diff
@@ -39,7 +39,7 @@ RUN chown -R peertube:peertube /app
 USER peertube
 
 RUN yarn install --pure-lockfile \
-    && npm run build -- $NPM_RUN_BUILD_OPTS \
+    && yarn run build -- $NPM_RUN_BUILD_OPTS \
```

Comment:
For consistency with the documentation, please keep using `npm`

Reply:
Done